.github/workflows/check.yml

```diff
@@ -1,19 +1,19 @@
 name: CI
 on:
   push:
-      paths:
-      - '**/Cargo.toml'
-      - '**/Cargo.lock'
-      - 'src/**/*'
-      - 'benches/**/*'
-      - 'tests/**/*'
+    paths:
+      - "**/Cargo.toml"
```

Comment:
wow, was it broken because of workspace introduction?

Reply:
It was because I missed specifying `in-memory-infra` to `cargo test`. I updated `check.yml` too because I think it needs `*/` in `paths` now.

Reply:
that was the question about single-quotes changed to double quotes :)